app/models/financial_planner.rb

```diff
@@ -1,2 +1,8 @@
 class FinancialPlanner < ApplicationRecord
+  before_save { self.email = email.downcase }
+  validates :name, presence: true, length: { maximum: 50 }
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+  validates :email, presence: true, length: { maximum: 255 }, format: { with: VALID_EMAIL_REGEX }, uniqueness: true
+  has_secure_password
+  validates :password, presence: true, length: { minimum: 6 }
```

Comment:
同じもの同士固めたほうがわかりやすいと思いました！

```suggestion
  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
  
  validates :name, presence: true, length: { maximum: 50 }
  validates :email, presence: true, length: { maximum: 255 }, format: { with: VALID_EMAIL_REGEX }, uniqueness: true
  validates :password, presence: true, length: { minimum: 6 }
  
  before_save { self.email = email.downcase }
  
  has_secure_password
```